--- gen-nodejs/Cassandra.js	2017-01-26 13:47:12.393570838 -0800
+++ Cassandra.js	2017-01-26 13:48:07.037693316 -0800
@@ -3,7 +3,7 @@
 //
 // DO NOT EDIT UNLESS YOU ARE SURE THAT YOU KNOW WHAT YOU ARE DOING
 //
-var Thrift = require('thrift').Thrift;
+var Thrift = require('helenus-thrift').Thrift;
 
 var ttypes = require('./cassandra_types');
 //HELPER FUNCTIONS AND STRUCTURES
@@ -516,7 +516,7 @@
   output.writeStructBegin('Cassandra_get_slice_args');
   if (this.key) {
     output.writeFieldBegin('key', Thrift.Type.STRING, 1);
-    output.writeString(this.key);
+    output.writeString(new Buffer(this.key, 'binary'));
     output.writeFieldEnd();
   }
   if (this.column_parent) {
@@ -3167,7 +3167,7 @@
       if (this.mutation_map.hasOwnProperty(kiter331))
       {
         var viter332 = this.mutation_map[kiter331];
-        output.writeString(kiter331);
+        output.writeString(new Buffer(kiter331, 'binary'));
         output.writeMapBegin(Thrift.Type.STRING, Thrift.Type.LIST, Thrift.objectLength(viter332));
         for (var kiter333 in viter332)
         {
@@ -9688,4 +9688,3 @@
     output.flush();
   })
 }
-
